augly/video/augmenters/ffmpeg/no_audio.py

```diff
@@ -1,22 +1,35 @@
 #!/usr/bin/env python3
 # Copyright (c) Facebook, Inc. and its affiliates.
 
-from typing import Tuple, Dict
+from typing import List, Optional
 
-from augly.video.augmenters.ffmpeg.base_augmenter import BaseFFMPEGAugmenter
-from ffmpeg.nodes import FilterableStream
+from augly.video.augmenters.ffmpeg.base_augmenter import BaseVidgearFFMPEGAugmenter
 
 
-class VideoAugmenterByRemovingAudio(BaseFFMPEGAugmenter):
-    def add_augmenter(
-        self, in_stream: FilterableStream, **kwargs
-    ) -> Tuple[FilterableStream, Dict]:
+class VideoAugmenterByRemovingAudio(BaseVidgearFFMPEGAugmenter):
+    def get_command(
+        self, video_path: str, output_path: Optional[str] = None, **kwargs
```

Comment:
Remove Optional everywhere like in https://github.com/facebookresearch/AugLy/pull/163